EIPS/eip-3404.md

```diff
@@ -0,0 +1,84 @@
+---
+eip: 3404
+title: Partial removal of refunds
+author: Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/eip-3298-removal-of-refunds/5430
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-16
+---
+
+## Simple Summary
+
+Overhaul SSTORE and SLOAD costs and storage refunds.
+
+## Motivation
```

Comment:
```suggestion
## Abstract

Adds tracking of cold state access and state modifications so that gas can be charged separately for cold vs hot reads and for reads vs writes.  Also introduces a mechanism for giving a minor incentive for contracts to leave slots with zero values rather than non-zero values.

## Motivation
```
An abstract would be nice here.  A terse human readable version of the specification, ideally a couple sentences long.